contrib/flesctl/init_run

```diff
@@ -82,7 +82,7 @@ for node in ${PN_LIST[@]}; do
     for i in `seq $TSCLIENT_COUNT`; do
         PARAM=${TSCLIENT_PARAM//%run_id%/$RUN_ID}
         PARAM=${PARAM//%client%/$i}
-        PN_SPM+=("$node: bash -c \"${SPMDIR}spm-require $SHM_PREFIX$output; ${FLESNETDIR}tsclient -c$output -s $SHM_PREFIX$output -L ${LOGDIR}${node}_${i}_tsclient.log --log-syslog ${PARAM}\"")
+        PN_SPM+=("$node: bash -c \"${SPMDIR}spm-require $SHM_PREFIX$output; ${FLESNETDIR}tsclient -c$output -i shm:$SHM_PREFIX$output -L ${LOGDIR}${node}_${i}_tsclient.log --log-syslog ${PARAM}\"")
```

Comment:
where ist the `//`?

Reply:
I left it out because I could not decide whether we want it there. The URI is parsed according to [rfc3986](https://www.rfc-editor.org/rfc/rfc3986#page-50). The `//` marks the next component as the "authority" (and not the "path"). I am not sure if this is more meant to be the hostname or something else that makes the URI globally unique. In this case, we really only use the a file name (or "identifier"), which can be the same on several systems. However, as I was not really sure about this, in the TimesliceAutoSource I append both authority and path and use the resulting string as shm_identifier.

Reply:
tl;dr: currently, it works with or without `//`. I am open to suggestions to change this.